src/main/java/li/cil/tis3d/api/ClientAPI.java

```diff
@@ -0,0 +1,14 @@
+package li.cil.tis3d.api;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+@Environment(EnvType.CLIENT)
+public final class ClientAPI {
+    public enum Font {
+        SmallFont,
+        NormalFont
+    }
+
+    public FontRendererAPI fontRendererAPI;
```

Comment:
`ClientAPI` feels a bit lonely, now that I had to move `ManualAPI` back out of it after discovering it's still intended as server side. The minimalist in me screams "just pass `FontRendererAPI` directly to the extensions!", though I am wary of future client API changes / additions. Maybe we could even move the manual API back here if it was intended to be client sided? (In reference to the documentation)

Reply:
I.. have no strong feelings about this one. I doubt anything is using the fact that the manual can be queried server-side for the existence of certain pages (which I think is what's there? I can't really remember). So.. maybe? I'll leave that to you :P